AddOns/BasicFinancials/test/src/TestBasicBF.Codeunit.al

```diff
@@ -0,0 +1,301 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+codeunit 139502 "Test Basic BF"
+{
+    Subtype = Test;
+    TestPermissions = Disabled;
+
+    var
+        Assert: Codeunit Assert;
+        AllProfileFilterTxt: Label 'MANUFACTURING|PROJECTS|SERVICES|WAREHOUSE|SHIPPING AND RECEIVING - WMS|SHIPPING AND RECEIVING|WAREHOUSE WORKER - WMS|PRODUCTION PLANNER|PROJECT MANAGER|DISPATCHER', Locked = true, Comment = 'As default the profile "SALES AND RELATIONSHIP MANAGER" cannot disable because it is set up as a default profile for one or more users or user groups.';
+        NotSupportedLicensesErr: Label 'Validation error for Field: IsSupportedLicenses,  Message = ''At least one user must have the Basic license.''', Locked = true;
+        NotSupportedCompaniesErr: Label 'Validation error for Field: IsSupportedCompanies,  Message = ''Exactly one company must exists in the environment.''', Locked = true;
+
+    trigger OnRun();
+    begin
+        // [FEATURE] [Basic] [User Experience]
+    end;
+
+    [Test]
+    procedure TestBasicExperienceTierAfterInstall();
+    var
+    begin
+        // [Scenario] Check the Basic Experience Tier After Install of Basic Extension 
+        // [Given] Application Area
+        // [WHEN]
+
+        // [THEN] Basic Experience Tier is Enabled
+        TestEnabledBasicExperienceTier();
+    end;
+
+    [Test]
+    procedure TestBasicApplicationAreaAfterInstall();
+    var
+    begin
+        // [Scenario] Check the Basic Application Area After Install of Basic Extension 
+        // [Given] Application Area
+        // [WHEN]
+
+        // [THEN] Basic Application Area is Enabled
+        TestEnabledBasicApplicationArea();
+    end;
+
+    [Test]
+    procedure TestDisableRoleCenterAfterInstall();
+    var
+    begin
+        // [Scenario] Check disable Role Center After Install of Basic Extension 
+        // [Given] Role Center
```

Comment:
// [GIVEN] Basic Experience extension installed
// [WHEN] Navigating to Role Centers
// [THEN] Set of defined Role Centers is disabled